design.qmd

```diff
@@ -33,58 +37,72 @@ The Analyst translates the scope for the analysis agreed with the Commissioner i
+## Documenting the design of the analysis
 
-On completion of the Design phase, the Assurer should be aware of the quality assurance tasks that will be required of them during the project lifetime and have assured the necessary elements of the Analytical plan.
+The design process should be documented in a proportionate manner. A design document that records the analytical plan should be produced by the Analyst, and signed-off by the commissioner. The design document may be reviewed by the Assurer.
 
-Iteration between the Commissioner and the Analyst is normal and expected whilst the analytical design develops. Considering analytical quality assurance and the acceptable margin of error for the Commissioner is essential to define how good is good enough. 
+It is best practice to use formal version control to track changes in the design document. 
 
-Agreeing the extent of the importance of the analysis output to decision making will set the level of [materiality](https://en.wikipedia.org/wiki/Materiality_(auditing)) (see also Definitions and Key Concepts), which will indicate what would be proportionate assurance.
 
-The Commissioner remains important through the analysis cycle, since they may well have greater expertise in the subject than the Analyst. Their contribution towards the input assumptions, data requirements and the most effective ways to present the outputs can prove invaluable.
+## Assuring the design phase
 
-The Analytical plan should be updated with formal version control to reflect revisions to the design including assurance activities.  
+The assurance of the design phase should consider whether the analytical plan:  
 
-## Documenting the design of the analysis
+* Delivers as intended
+* Adequately addresses the complexities of the customer issue for this purpose 
+* Is well-structured for the purpose, data driven, and reflects a robust overall design.
 
-Best practice is to document the design process recording how the proposed analytical process generates the requested insights. This plan should be supported by the design documentation.
+The assurance of the design may be carried out by the Assurer. For more complex analysis, it is good practice to engage subject matter experts to provide independent assurance, and to ensure the accuracy and limitations of the chosen methods are understood, ideally with tests baselining their response against independent reference cases.
 
-## Analysing uncertainty
+
+
+ 
+
+
+
+
+## Treatement of uncertainty in the design phase
 
 During the design phase, Analysts should examine the planned analysis systematically for possible sources and types of uncertainty, to maximise the chance of identifying all that are sufficiently large to breach the acceptable margin of error.
```

Comment:
Think we need to think about how we talk about what is an acceptable level of uncertainty. IN my experience most commissioners find specifying an acceptable level of uncertainty very difficult.